Split schema title and version

nwlandry opened the pull request:
This PR creates two fields from the title to specify the name and the version. In addition, the file has been renamed, so that by default, users will always pull from the most recent version.

nwlandry commented:
I decided to rename the file `schema.json` for brevity.

colltoaction reviewed (commented):
Is this related to the Schema Change Process you're writing about? We could use v0.0.1 as the first item in the changelog.

nwlandry commented:
So you're saying that I could copy `schema.json` into `schema_dev.json` and `schema_v0.0.1.json`?

colltoaction commented:
Yes, that seems to be correct. And creating a CHANGELOG.md file. What do you think?

nwlandry commented:
There's a changelog [here](https://github.com/pszufe/HIF-standard/blob/split-name-with-version/schemas/CHANGELOG.md)---do you think that it should be moved to a more visible place?

colltoaction commented:
> There's a changelog [here](https://github.com/pszufe/HIF-standard/blob/split-name-with-version/schemas/CHANGELOG.md)---do you think that it should be moved to a more visible place?

Ah I missed it! It's great. I checked the README and noticed there was a typo in the URL.

nwlandry commented:
Okay, I made the suggested changes! Thanks for the input.

colltoaction reviewed (approved):
Note that `0.0` doesn't follow [semver](https://semver.org/) but the rest looks good. Consider reverting to `0.1.0`.

nwlandry commented:
> Note that `0.0` doesn't follow [semver](https://semver.org/) but the rest looks good. Consider reverting to `0.1.0`.

Thanks for noticing this! Fixed. Two things: (1) I deleted the file with the version in the name because it doesn't follow the process that I outlined in the manuscript and (2) I changed the version in the dev schema to "dev". That way, each file has a unique version and there are no duplicates.

nwlandry commented:
Okay, after thinking more about it, perhaps it would be more streamlined to remove `schema_dev.json`, change the version in `schema.json` to `latest` and then when we feel that it's ready for a stable version, we could copy and name it with a version. How does this sound, @colltoaction?

colltoaction commented:
> Okay, after thinking more about it, perhaps it would be more streamlined to remove `schema_dev.json`, change the version in `schema.json` to `latest` and then when we feel that it's ready for a stable version, we could copy and name it with a version. How does this sound, @colltoaction?

I understand that with this process we would now have `schema.json` and `schema_v0.1.0.json`. SGTM.

nwlandry commented:
> > Okay, after thinking more about it, perhaps it would be more streamlined to remove `schema_dev.json`, change the version in `schema.json` to `latest` and then when we feel that it's ready for a stable version, we could copy and name it with a version. How does this sound, @colltoaction?
> 
> I understand that with this process we would now have `schema.json` and `schema_v0.1.0.json`. SGTM.

yep! that's right. I will merge now.
